script/apps/WebServer/admin/wp-content/pages/wiki/mod/packages/models/packages.page

```diff
@@ -135,23 +168,27 @@ function packages:api_getOnePackage(params)
 end
 
 function packages:api_modifyPackage(params)
+	self:ensureAuthenticated();
+	local err, query = self:validateQuery(params);
+
+	if(not self:getUserId()) then
+		return {result = "-1",msg = "This API must be authentication"};
+	end
+
 	data = {};
 
 	self:db():updateOne(
-		{_id = params.packageId},
-		{projectName     = params.projectName,
-		 projectDesc     = params.projectDesc,
-		 projectGitURL   = params.projectGitURL,
-		 projectUpdate	= ParaGlobal.GetDateFormat("yyyy-MM-dd"),
-		 projectReleases = params.projectReleases,
-		 version         = params.version},
-		function(err, response)
-			data = response;
-			resume();
-		  --log({"modifyPackage",data}) 
-		end
+		{_id = query.packageId},
```

Comment:
u need to add {userid=self:GetUserId()} to prevent people to modify others's package